src/cdk/tree/tree.ts

```diff
@@ -321,7 +321,7 @@ export class CdkTree<T, K = T> implements AfterContentChecked, CollectionViewer,
   exportAs: 'cdkTreeNode',
   host: {
     'class': 'cdk-tree-node',
-    '[attr.aria-expanded]': 'isExpanded',
+    '[attr.aria-expanded]': 'isLeafNode',
```

Comment:
I think it makes more sense to create a new function `isLeafNode` to use in conjunction with this conditional:

`isLeafNode() ? null : isExpanded`

RIght now, it's not obvious why `isLeafNode` is returning the expansion state rather than just a `true` or `false` of whether it's a leaf node

Reply:
Added recommended changes, thanks!